### PR TITLE
text def stomach neuroendocrine cell 

### PR DESCRIPTION
**Pull request by JABelfiore**
def added, synonyms added

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -24989,6 +24989,9 @@ EquivalentClasses(obo:CL_1000217 ObjectIntersectionOf(obo:CL_0000138 ObjectSomeV
 
 # Class: obo:CL_1000222 (stomach neuroendocrine cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") obo:IAO_0000115 obo:CL_1000222 "A specialised neuroendocrine cell located in the gastric mucosa that regulates digestive processes including acid secretion and motility. This cell stores hormones in large dense core vesicles and synaptic-like microvesicles.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") oboInOwl:hasBroadSynonym obo:CL_1000222 "DES cell")
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:24993]: Consider adding the Xref as PMID 
PMID:15153415 for this reference

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -24989,6 +24989,9 @@ EquivalentClasses(obo:CL_1000217 ObjectIntersectionOf(obo:CL_0000138 ObjectSomeV
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") oboInOwl:hasBroadSynonym obo:CL_1000222 "diffuse endocrine system cell")
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:24994]: Consider adding the Xref as PMID
PMID:15153415 for this reference

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -24989,6 +24989,9 @@ EquivalentClasses(obo:CL_1000217 ObjectIntersectionOf(obo:CL_0000138 ObjectSomeV
...
 AnnotationAssertion(rdfs:label obo:CL_1000222 "stomach neuroendocrine cell")
 EquivalentClasses(obo:CL_1000222 ObjectIntersectionOf(obo:CL_0000165 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000945)))
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:24997]: Consider adding 'capable of' some 'acid secretion'

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -24989,6 +24989,9 @@ EquivalentClasses(obo:CL_1000217 ObjectIntersectionOf(obo:CL_0000138 ObjectSomeV
 
 # Class: obo:CL_1000222 (stomach neuroendocrine cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") obo:IAO_0000115 obo:CL_1000222 "A specialised neuroendocrine cell located in the gastric mucosa that regulates digestive processes including acid secretion and motility. This cell stores hormones in large dense core vesicles and synaptic-like microvesicles.")
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:24992]: do you mean gut motility or cell motility? 
if it is cell motility include 'capable of' some 'cell motility' in the logical definition

**Review by AvolaAmg — CHANGES_REQUESTED** [review]
Please see in-line comments

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:24993]: Amended

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:24994]: Amended

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:24997]: Added

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:24992]: Gut motility, updated def to reflect this

**Review by AvolaAmg — APPROVED** [review]

**Pull request merged**